scenarios/detection/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -41,14 +41,17 @@
   {
```

Comment:
<p>this image doesn't reflect what new 'data sub folder'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Can you clarify what you mean? The content that's shown (the annotation and images directories) is inside the data_subfolder